torch/testing/_internal/common_methods_invocations.py

```diff
@@ -8647,6 +8647,7 @@ def ref_pairwise_distance(input1, input2):
                # https://github.com/pytorch/pytorch/issues/71784
                DecorateInfo(unittest.skip('Skipped!'), 'TestNNCOpInfo', 'test_nnc_correctness',
                             device_type='cpu', dtypes=(torch.float16,)),
+               DecorateInfo(unittest.skip('Skipped!'), 'TestCudaFuserOpInfo', 'test_nvfuser_correctness', dtypes=(torch.float16,)),
```

Comment:
Cool! should we file issues for the failing tests ?

Reply:
I think most of them either have an issue filed or are expected to fail

e.g. https://github.com/pytorch/pytorch/issues/71784 for this one

Reply:
Let me jump on the failing tests!

Reply:
@jjsjann123 fyi I think #71784 might be expected

And list of tests that need fixes is in #75029 (also see this board: https://github.com/pytorch/pytorch/projects/30)